### PR TITLE
Remove sqlite3 dependency, broken SQLite support

### PR DESCRIPTION
**Pull request by myabc**
We do not test against, or officially support, SQLite.

This PR has the same intention as #1167, but a narrower scope.

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://feature-remove-sqlite3-dependency.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

File: test/unit/lib/chili_project/database_test.rb [hunk 1]
```diff
@@ -30,71 +30,16 @@
 
 class ChiliProject::DatabaseTest < ActiveSupport::TestCase
   setup do
-    ChiliProject::Database.stub(:adapter_name).and_return "SQLite"
+    ChiliProject::Database.stub(:adapter_name).and_return "PostgresQL"
```
Comment (hschink) [inline test/unit/lib/chili_project/database_test.rb]: @myabc I would merge this as soon as this is 'fixed'.

**Comment by myabc** [comment]
@kgalli Can we also remove the references to `libsqlite3-dev` from the [tutorial here](https://github.com/opf/openproject/blob/feature/remove-sqlite3-dependency/doc/operation_guides/manual/ubuntu-trusty-14-04-lts-and-debian-wheezy-7-7-production.md)?

**Comment by ulferts** [comment]
I merged dev in, let's see if I made a mistake.

There are still sqlite references e.g. in environments/(production|development).rb which are just comments but in the install instruction as well (https://github.com/opf/openproject/blob/dev/doc/operation_guides/manual/ubuntu-trusty-14-04-lts-and-debian-wheezy-7-7-production.md). All of those could be removed IMO.

**Comment by myabc** [comment]
>  in the install instruction as well (https://github.com/opf/openproject/blob/dev/doc/operation_guides/manual/ubuntu-trusty-14-04-lts-and-debian-wheezy-7-7-production.md). All of those could be removed IMO.

I was waiting for a response from @kgalli to https://github.com/opf/openproject/pull/2204#issuecomment-63271521. I guess we should test the install tutorial.

**Comment by kgalli** [comment]
@myabc In my opinion we can remove `libsqlite3-dev` and yes we have to update the install tutorial as well. I will have a look at the tutorials tomorrow and make the necessary adjustments.

**Comment by ulferts** [comment]
@kgalli any updates on this one?

**Pull request merged**